workspace/src/generate-benchmark-report.js

```diff
@@ -7,13 +7,22 @@ import * as url from 'url'
 import {promises as fs, existsSync} from 'fs'
 import {createRequire} from 'module'
 import {execa} from 'execa'
+import {osMetadata} from './os-metadata.js'
 
 const require = createRequire(import.meta.url)
 const rootDirectory = path.join(url.fileURLToPath(new URL('.', import.meta.url)), '../..')
 
 const baselineBenchmarkPath = path.join(path.join(rootDirectory, 'baseline-benchmark.json'))
 const benchmarkPath = path.join(rootDirectory, 'benchmark.json')
 
+function getOSMarkdownReport(osMetadata) {
```

Comment:
What about showing the current CPU/memory usage as well? The load while benchmarking could be different and affect as well, no?

Maybe this helps: https://labs.quansight.org/blog/2021/08/github-actions-benchmarks

Reply:
Yeah. I minimized it a bit by running it a few times and obtaining the average. It's much better than what we had before.